packages/contracts/tasks/l2-gasprice.ts

```diff
@@ -0,0 +1,64 @@
+/* Imports: External */
+import { ethers } from 'ethers'
+import { task } from 'hardhat/config'
+import * as types from 'hardhat/internal/core/params/argumentTypes'
+
+import { predeploys } from '../src/predeploys'
+import { getContractDefinition } from '../src/contract-defs'
+
+task('set-l2-gasprice')
+  .addOptionalParam('l2GasPrice', 'Gas Price to set on L2', 0, types.int)
+  .addOptionalParam(
+    'transactionGasPrice',
+    'Gas price to set on L2 transaction',
```

Comment:
Maybe a typo here? Confusing wording to me.

Reply:
Would it be more clear if i said it was `tx.gasPrice`? It is the gas price that is set on the transaction being sent to L2

Reply:
Ahhh I see. Yeah explicitly saying `tx.gasPrice` is more clear.